cli/cmds.go

```diff
@@ -615,3 +677,46 @@ func upgrade012File(fs afero.Fs, log *logrus.Logger, filename string, fmtverbs,
 
 	return &br, err
 }
+
+func grepInFile(fs afero.Fs, log *logrus.Logger, filename string, verbose bool, opts []hclgrep.Option, stdin io.Reader, stdout, stderr io.Writer) (*blocks.Reader, error) {
+	var blockWriter blocks.BlockWriter
+	blockWriter = textBlockWriter{
+		writer: stdout,
+	}
+
+	br := blocks.Reader{
+		Log:         log,
+		ReadOnly:    true,
+		LineRead:    blocks.ReaderIgnore,
+		BlockWriter: blockWriter,
+		BlockRead: func(br *blocks.Reader, i int, b string) error {
+			b = verbs.Escape(b)
+			rbuf := strings.NewReader(strings.Repeat("\n", br.LineCount-br.BlockCurrentLine) + b)
+			wbuf := bytes.NewBuffer([]byte{})
+			m := hclgrep.NewMatcher(append(opts, hclgrep.OptionOutput(wbuf))...)
+			if err := m.File(filename, rbuf); err != nil {
+				return err
+			}
+			if wbuf.String() == "" {
+				return nil
+			}
+			br.BlockWriter.Write(br.BlockCount, br.LineCount-br.BlockCurrentLine, br.LineCount, verbs.Unscape(wbuf.String()))
```

Comment:
please use new lines for readability 
```suggestion

			m := hclgrep.NewMatcher(append(opts, hclgrep.OptionOutput(wbuf))...)
			if err := m.File(filename, rbuf); err != nil {
				return err
			}
			
			if wbuf.String() == "" {
				return nil
			}
			
			br.BlockWriter.Write(br.BlockCount, br.LineCount-br.BlockCurrentLine, br.LineCount, verbs.Unscape(wbuf.String()))
```